Sources/App/Websocket/RoomUpdate.swift

```diff
@@ -0,0 +1,15 @@
+import Vapor
+
+struct RoomUpdate: Content {
+    enum UpdateType: Int, Codable {
```

Comment:
Probably should move this out of the RoomUpdate scope?

Reply:
just move to top of class